hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/encryption.md

```diff
@@ -727,6 +736,28 @@ S3-CSE to work.
  </property>
 ```
 
+#### 2. CSE-CUSTOM
+- Set `fs.s3a.encryption.algorithm=CSE-CUSTOM`.
+- Set `fs.s3a.encryption.cse.custom.cryptographic.material.manager.class.name=<fully qualified class name>`.
```

Comment:
Should the new `fs.s3a.encryption.cse` properties be added to `core-default.xml` and `index.md`?

Reply:
These configurations will be shown as part of the encryption page. similar to https://hadoop.apache.org/docs/r3.4.0/hadoop-aws/tools/hadoop-aws/encryption.html